docs/pages/web/sheetmobile.js

```diff
@@ -0,0 +1,326 @@
+              />
+            }
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Footer"
+          description={`SheetMobile's footer has a flexible configuration. \`footer\` prop accepts any kind of node. The footer can have up to two Buttons and another two IconButtons as shown in the example.`}
+        >
+          <MainSection.Card
+            cardSize="lg"
+            sandpackExample={
+              <SandpackExample code={footer} name="Footer example" layout="column" />
+            }
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Animation"
+          columns={2}
+          description={`By default, SheetMobile animates *in* (*up*), with the initial render process from the entry-point, and *out* (*down*), when the \`ESC\` key is pressed, the header close button is pressed, or the user clicks outside of the SheetMobile. However, to trigger the exit-animation from other elements in other areas such as the \`children\` or \`footer\`, the following render prop can be used:
+
+~~~jsx
+<SheetMobile.DismissingElement>
+  ({ onDismissStart }) => ( ... )
+</SheetMobile.DismissingElement>
+~~~
+
+For exposed \`onClick\` props where it's not possible to use SheetMobile.DismissingElement, the \`onClick\` passes "onDismissStart" as render props along with the event.
+
+~~~jsx
+onClick={({ onDismissStart }) => onDismissStart() }
+~~~
+
+When using these render props, just pass the argument \`onDismissStart\` to your exit-point action elements or execute them on the \`onClick\`. In the example below, we've added the exit animation to the:
+- Back arrow icon button (header)
+- "Close" gray button (children)
+- "Close" red button (footer)
+`}
+        >
+          <MainSection.Card
+            cardSize="lg"
+            sandpackExample={
+              <SandpackExample code={animation} name="Dismiss animation example" layout="column" />
+            }
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Preventing close on outside click"
+          description={` By default, users can click outside a partial SheetMobile (on the overlay) to close it. This can be disabled by setting \`closeOnOutsideClick\` to "false".
+`}
+        >
+          <MainSection.Card
+            cardSize="lg"
+            sandpackExample={
+              <SandpackExample
+                code={outsideClick}
+                name="Preventing outside click dismissal example"
+                layout="column"
+              />
+            }
+          />
+        </MainSection.Subsection>
+      </MainSection>
+
+      <MainSection name="Writing">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description={`
+- Keep headings short and clear
+- Use Sentence case for headings per our Pinterest writing standards
```

Comment:
This should link to the Content Guidelines page

Reply:
DONE